api/v1/registry.proto

```diff
@@ -111,20 +112,55 @@ message Parameter {
     // Description of what the effects of the parameter are.
     string description = 2;
     // Value of the parameter
```

Comment:
added type as an enum since all values must ultimately be strings. How those strings are represented until they reach the renderer will be left as an implementation detail for its consumers.